api/v1alpha3/backup/backup.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2024 VMware, Inc. All Rights Reserved.
```

Comment:
Hmm.  I meant a different "file" not a pacakge.  And if we are considering a different package, should we move it out of the API schema to make sure that it is independent of the schema versions?